python/dgl/distributed/dist_context.py

```diff
@@ -44,7 +44,23 @@ def _init_rpc(ip_config, num_servers, max_queue_size, net_type, role, num_thread
 def initialize(ip_config, num_servers=1, num_workers=0,
                max_queue_size=MAX_QUEUE_SIZE, net_type='socket',
                num_worker_threads=1):
-    """Init rpc service
+    """Initialize DGL's distributed module
+
+    This function has to be invoked at the very beginning of the training script.
+    For example, when used with Pytorch, this function has to be invoked before
+    Pytorch's `pytorch.distributed.init_process_group`.
+
+    The initialization function acts differently in different modes. When the training
+    scripts runs as a trainer process, this API builds connections with DGL servers and
+    creates sampler processes; when the training script runs as a server process,
+    this API runs the server code and never returns.
+
+    DGL uses multiprocessing to parallelize distributed sampling. The sampling processes
+    have to be created in advance. `num_workers` specifies the number of sampling worker
```

Comment:
```suggestion
    DGL uses multiprocessing to parallelize distributed sampling. You must create the sampling
    processes in advance. `num_workers` specifies the number of sampling worker
```

Also what does `in advance` precisely mean?  Maybe providing a link to a user guide/tutorial/example is helpful.